composer.json

```diff
@@ -53,5 +54,17 @@
     },
     "config": {
         "sort-packages": true
+    },
+    "scripts": {
+        "phpunit": "phpunit --colors=always --verbose",
+        "php-cs-fixer": "php-cs-fixer fix -v --dry-run",
+        "quality": [
+            "@phpstan",
+            "@phpinsights"
+        ],
+        "test": [
+            "@php-cs-fixer",
```

Comment:
This command should not contain the argument `fix`.

Reply:
with the `--dry-run` option it won't fix automatically errors. That's what was proposed in https://github.com/algolia/algoliasearch-client-php/pull/560/files for example